releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,33 @@
+---
+critical:
+  - |
+    From this release on, a rcbops fork is used for the openstack-ansible
+    submodule. Due to this change, a pre-existing checkout of rpc-openstack will
+    require a submodule sync to ensure that the subodule is pointing to the
+    right repository. Failing to keep the submodule in sync will result in some
+    unexpected behavior, primarily you will be missing any code updates that
+    have been pushed up to the forked repository. To resolve this execute the
```

Comment:
nit: I think this is probably unnecessary, making this all a bit wordy:

``Failing to keep the submodule in sync will result in some unexpected behavior, primarily you will be missing any code updates that have been pushed up to the forked repository.``